common/config/rush/command-line.json

```diff
@@ -11,6 +11,16 @@
             "ignoreMissingScript": false,
             "allowWarningsInSuccessfulBuild": true
         },
+        {
+            "commandKind": "bulk",
+            "name": "mintest",
```

Comment:
Adding a new test pass which runs the tests agains the minified version of the core code.